tox.ini

```diff
@@ -67,8 +67,8 @@ commands =
 description = Run integration tests
 deps =
     jinja2
-    #git+https://github.com/juju/python-libjuju.git
-    juju
+    git+https://github.com/juju/python-libjuju.git
```

Comment:
What’s with this, this time? :-)

Reply:
`block_until_with_coroutine` is not released yet.